extensions/amp-story/1.0/test/validator-amp-story-amp-render.html

```diff
@@ -0,0 +1,98 @@
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  Tests for the amp-render functionality inside amp-story.
+-->
+<!DOCTYPE html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story"
+      src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-render"
+      src="https://cdn.ampproject.org/v0/amp-render-1.0.js"></script>
+    <script async custom-template="amp-mustache"
+      src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+    <script async custom-element="amp-bind"
+      src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <title>amp-render</title>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1"/>
+    <link rel="canonical" href="index.html" />
+    <style amp-boilerplate>
+      body{
+        -webkit-animation:-amp-start 8s steps(1, end) 0s 1 normal both;
+        -moz-animation:-amp-start 8s steps(1, end) 0s 1 normal both;
+        -ms-animation:-amp-start 8s steps(1, end) 0s 1 normal both;
+        animation:-amp-start 8s steps(1, end) 0s 1 normal both
+      }
+      @-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+      @-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+      @-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+      @-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+      @keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}
+    </style>
+    <noscript>
+      <style amp-boilerplate>
+        body{
+          -webkit-animation:none;
+          -moz-animation:none;
+          -ms-animation:none;
+          animation:none
+        }
+      </style>
+    </noscript>
+  </head>
+
+  <body>
+    <amp-story standalone title="My Story" publisher="Me"
+      publisher-logo-src="http://me.com/logo.png"
+      poster-portrait-src="http://me.com/poster.jpg">
+      <amp-story-page id="page-rendering">
+        <amp-story-grid-layer template="vertical">
+          <amp-render
+            layout="fixed-height"
+            height="100"
+            src="/examples/images.json"
+            binding="no">
+            <template type="amp-mustache">
+              <div><a href="{{url}}">{{title}}</a></div>
```

Comment:
Plz use a working example, like
```
          <template type="amp-mustache" id="my-template">
            <div>{{#images}}<a href="{{url.unsplash}}">{{author}}</a>{{/images}}</div>
          </template>
```

Reply:
Thanks! I've updated the PR with what I believe is a working example, but viewing `validator-amp-story-amp-render.html` on a local server only shows the "Loading ..." placeholder along with the following error logged to the console:
`amp-render source must start with "https://" or "//" or be relative and served from either https or from localhost. Invalid value: /examples/images.json`

It's unclear to me why this path is invalid, though I'm unsure whether this is actually an issue because I notice that some of the other validator HTML test files also produce file-loading errors (e.g., `validator-amp-story-360.html`).